src/libraries/Common/src/System/Net/Security/NegotiateStreamPal.Unix.cs

```diff
@@ -341,6 +236,22 @@ Interop.NetSecurityNative.Status status
                 if (NetEventSource.Log.IsEnabled()) NetEventSource.Error(null, ex);
                 return new SecurityStatusPal(SecurityStatusPalErrorCode.InternalError, ex);
             }
+            finally
+            {
+                token.Dispose();
+
+                // Save the inner context handle for further calls to NetSecurity
+                //
+                // For the first call `negoContext.GssContext` is invalid and we expect the
+                // inital handle to be returned from AcceptSecContext. For any subsequent
+                // call the handle should stay the same or it can be destroyed by the native
+                // AcceptSecContext call.
```

Comment:
I should update the comment here to mention `InitSecContext` instead of `AcceptSecContext`.